Additional methods for Folding

ivanslabbert opened the pull request:
Hi,

I added the `FoldChildren(int line, FoldAction action)` and `GetFoldParent(int startLine)` methods. The `FoldAll(FoldAction.Contract)` method only contracts the top-level folder (from what I can see this is handled by the native Scintilla dll?). By using the `FoldChildren()` on the first fold line, it is possible to contract all folders.

Btw, thanks for the excellent work with the wrapper.

Regards,
Ivan Slabbert

jacobslusser commented:
Thanks.

jacobslusser commented:
I reverted the merge because I just realized that we already have these implemetned. See `Line.FoldChildren(FoldAction)` and `Line.FoldParent`.

Thanks though. :)

ivanslabbert commented:
No problem. I wasn’t aware that it already was implemented. I changed my code to use the Line.FoldChildren and Line.FoldParent.

Thanks


From: jacobslusser [mailto:notifications@github.com]
Sent: 29 July 2015 06:47 AM
To: jacobslusser/ScintillaNET <ScintillaNET@noreply.github.com>
Cc: Ivan Slabbert <Ivan@staro.co.za>
Subject: Re: [ScintillaNET] Additional methods for Folding (#89)


I reverted the merge because I just realized that we already have these implemetned. See Line.FoldChildren(FoldAction) and Line.FoldParent.

Thanks though. :)

—
Reply to this email directly or view it on GitHub<https://github.com/jacobslusser/ScintillaNET/pull/89#issuecomment-125838848>.
